src/views/ManageTeamHealth.vue

```diff
@@ -266,6 +317,44 @@ export default {
 </template>
 
 <style>
+.popover__criteria {
+  display: block;
+  align-items: left;
+  background: white;
+  border: solid 1px var(--light-grey);
+  color: var(--dark-grey);
+  max-width: 470px;
+  font-family: "Avenir";
+  font-size: 15px;
+  box-shadow: 3px 3px 3px var(--grey);
+}
+
+.popover__criteria h4 {
```

Comment:
Any way we can get SASS?

Reply:
I think initially it seemed overkill, but there's no reason we can't add it now.